charts/castai-evictor/values.yaml

```diff
@@ -4,44 +4,44 @@
 
 replicaCount: 1
 
-# labels to add to all resources
+# commonLabels -- Labels to add to all resources.
 commonLabels: {}
 
-# annotations to add to all resources
+# annotations - Annotations to add to all resources.
 commonAnnotations: {}
 
-# Specifies whether the Evictor should run in dryRun mode (Read-Only).
+# dryRyn -- Specifies whether the Evictor should run in dryRun mode (Read-Only).
 # if true, evictor will just log action(s) it would perform
 # instead of actually performing them.
 dryRun: false
 
-# Specifies whether the Evictor can behave as aggressive
+# aggressiveMode -- Specifies whether the Evictor can behave as aggressive
```

Comment:
We should probably stay consistent and either add period at the end of each value description or none of them